docs/articles/kind-deploy.md

```diff
@@ -0,0 +1,220 @@
+# Using kind for Deployment Testing
```

Comment:
`kind` so it stands out

Reply:
Same below everywhere

Reply:
Do you mean to highlight the word 'kind' such as bold or code notation?  That can be done throughout, although not in the title itself.